libautoscoper/src/PSO.cpp

```diff
@@ -38,7 +38,7 @@ void intializeRandom()
 
 float getRandom(float low, float high)
 {
-  return low + (high - low + 1.0f)*(float)rand() / (RAND_MAX + 1.0f);
+  return low + (high - low) * getRandomClamped();
```

Comment:
This is consistent with implementation of `vtkMinimalStandardRandomSequence::GetRangeValue(double rangeMin, double rangeMax)`[^1]

[^1]: https://github.com/Kitware/VTK/blob/fe0294298f14539349d251188984ab36a262d64b/Common/Core/vtkMinimalStandardRandomSequence.cxx#L85-L100